schemas/v1.0/azure.yaml.json

```diff
@@ -148,6 +148,22 @@
                 },
                 "required": ["project"]
             }
+        },
+        "pipeline": {
+            "type": "object",
+            "title": "Definition of continuous integration pipeline",
+            "additionalProperties": true,
```

Comment:
I would have expected that this should be `false`, not `true`?  what are the additional properties you could set and what interprets them?